src/sentry/static/sentry/app/views/settings/organization/stats/projectTable.jsx

```diff
@@ -1,107 +1,91 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import styled from 'react-emotion';
 import {Link} from 'react-router';
 
+import {ProjectTableLayout, ProjectTableDataElement} from './projectTableLayout';
 import Count from '../../../../components/count';
-import {t} from '../../../../locale';
+import space from '../../../../styles/space';
 
-let getPercent = (item, total) => {
-  if (total === 0) {
-    return '';
-  }
-  if (item === 0) {
-    return '0%';
-  }
-  return parseInt(item / total * 100, 10) + '%';
-};
-
-class ProjectTable extends React.PureComponent {
-  static propTypes = {
-    projectMap: PropTypes.object.isRequired,
-    projectTotals: PropTypes.array.isRequired,
-    orgTotal: PropTypes.object.isRequired,
-    organization: PropTypes.object.isRequired,
+const ProjectTable = ({projectMap, projectTotals, orgTotal, organization}) => {
```

Comment:
this component has no mixins and no internal state, so a pure function seemed like the right thing here. lmk if that is wrong and i am dum.

Reply:
I would keep what you have, but imo, I stick to class components because 1) it's annoying going from functional -> class when you need to add state 2) afaik there are no performance benefits to functional components (yet), 3) can always change them to PureComponents

Reply:
ok for a while we were doing quite a few pure class components, so this is good to know.